standards/typescript/tslint/Degordian code style.xml

```diff
@@ -0,0 +1,235 @@
+<code_scheme name="Degordian code style" version="173">
+  <option name="LINE_SEPARATOR" value="&#xD;&#xA;" />
+  <TypeScriptCodeStyleSettings>
+    <option name="SPACE_BEFORE_FUNCTION_LEFT_PARENTH" value="false" />
+    <option name="ENFORCE_TRAILING_COMMA" value="WhenMultiline" />
+    <option name="SPACES_WITHIN_IMPORTS" value="true" />
+    <option name="IMPORT_SORT_MODULE_NAME" value="true" />
+    <option name="IMPORT_PREFER_ABSOLUTE_PATH" value="FALSE" />
+  </TypeScriptCodeStyleSettings>
+  <codeStyleSettings language="PHP">
```

Comment:
Sorry, this is also php :D

Reply:
my bad, I updated storm and had to update some shortcuts to didn't update it till the end haha